app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = Like.new(author_id: current_user.id, post_id: @post.id)
```

Comment:
- [ ] I would suggest here to make use of the current user that you have created on application controller, it is a good practice 

```
      @like = @current_user.likes.new(post_id: @post.id)
```